fighthealthinsurance/common_view_logic.py

```
@@ -1659,7 +1656,11 @@
         except Exception as e:
             logger.debug(f"Error gathering contexts: {e}")
             # We still might have saved a context.
-            await denial.arefresh_from_db()
+            try:
+                # Added in Django 5.1
+                await denial.arefresh_from_db(from_queryset=denial_query)
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)